plugins/woocommerce/client/admin/client/wp-admin-scripts/product-tracking/shared.ts

```diff
@@ -181,21 +180,36 @@ export const getProductData = () => {
  * @return string
  */
 const getPublishDate = ( prefix = '' ) => {
-	const month = (
-		document.querySelector( `#${ prefix }mm` ) as HTMLInputElement
-	 )?.value;
-	const day = (
-		document.querySelector( `#${ prefix }jj` ) as HTMLInputElement
-	 )?.value;
-	const year = (
-		document.querySelector( `#${ prefix }aa` ) as HTMLInputElement
-	 )?.value;
-	const hours = (
-		document.querySelector( `#${ prefix }hh` ) as HTMLInputElement
-	 )?.value;
-	const seconds = (
-		document.querySelector( `#${ prefix }mn` ) as HTMLInputElement
-	 )?.value;
+	const month =
```

Comment:
These changes reduce the chances of `undefined` appearing as text in the returned string